seqr/views/apis/staff_api.py

```diff
@@ -409,6 +409,25 @@ def discovery_sheet(request, project_guid):
     })
 
 
+@staff_member_required(login_url=API_LOGIN_REQUIRED_URL)
+def success_story(request, success_story_types):
+    errors = []
+
+    if success_story_types == 'all':
+        all_families = Family.objects.filter(~Q(success_story=None))
+    else:
+        all_families = Family.objects.none()
+        for success_story_type in success_story_types:
+            families = Family.objects.filter(success_story_types__contains=[success_story_type])
+            all_families = all_families.union(families)
+    rows = _generate_family_rows(all_families)
```

Comment:
The helper function is really small and only used once, so I think its more readable just to do it in line here:
```
rows = [{
     "project_guid": family.project.guid,
     "family_guid": family.guid,
     "family_id": family.family_id,
     "success_story_types": family.success_story_types,
      "success_story": family.success_story,
      "row_id": family.guid,
} for family in all_families]
```